### PR TITLE
Small fixes, improving logging

### PR DESCRIPTION
**Pull request by CsilLAB**
- renamed run_time to time_elapses, as run_time is also used in the Lambda payload
and also in locust and it was sightly confusing
- added get_ prefix to getter methods
- replaced log_ prefix with increase_ to clarify the intent
- added some log messages and output to help understand what's happening
- fixed some typos in comments
- added object variables for better control of error thresholds
- updated readme example outputs
- bumped version

### CODE REVIEW

**Comment by CsilLAB** [comment]
Hey @FutureSharks, before I started working on the compatibility with locust 1.0.3, I went though some of the code to make sure I understand it well. These changes helped me read the code, I hope you find them beneficial as well.

File: README.md [hunk 1]
```diff
@@ -182,92 +182,93 @@ load_test = LambdaLoadTest(
...
-INFO:invokust.aws_lambda.lambda_load_test:Invocation complete. Requests (errors): 20 (0), execution time: 3037, sleeping: 0
-INFO:root:threads: 2, rpm: 773, run_time: 30, requests_total: 340, request_fail_ratio: 0.0, invocation_error_ratio: 0.0
-INFO:invokust.aws_lambda.lambda_load_test:Invocation complete. Requests (errors): 20 (0), execution time: 3078, sleeping: 0
-INFO:invokust.aws_lambda.lambda_load_test:Invocation complete. Requests (errors): 20 (0), execution time: 3100, sleeping: 0
-INFO:root:threads: 2, rpm: 758, run_time: 33, requests_total: 380, request_fail_ratio: 0.0, invocation_error_ratio: 0.0
-INFO:root:Time limit reached
-INFO:root:Waiting for threads to exit...
-INFO:invokust.aws_lambda.lambda_load_test:Invocation complete. Requests (errors): 20 (0), execution time: 2965, sleeping: 0
+INFO:root:Invoking lambda...
+INFO:root:threads: 2, rpm: 0, time elapsed: 6s, total requests from finished threads: 0, request fail ratio: 0, invocation error ratio: 0
```
Comment (FutureSharks) [inline README.md:203]: How come there's no `rpm` numbers in here?

Comment (FutureSharks) [reply README.md:203]: Meh, never mind, I see it's showing in the other output

**Comment by FutureSharks** [comment]
@CsilLAB do you want me to merge and release this now?

File: README.md (hunk 1 above)
Comment (CsilLAB) [reply README.md:203]: I am glad you asked...I spent about an hour trying to figure it out :D

The reason is that the total rpm and all the other total stats only gets updated when a lambda finished. That's why I changed the `requests` -> `total requests from finished threads` to make it clearer. Probably could have done for the rpm too.

**Comment by CsilLAB** [comment]
Yes please, a release would be awesome!

**Comment by FutureSharks** [comment]
Done: https://pypi.org/project/invokust/0.72/

**Comment by CsilLAB** [comment]
Thank you so much!

**Pull request merged**